smart-contracts/contracts/Unlock.sol

```diff
@@ -135,6 +143,17 @@ contract Unlock is UnlockInitializable, UnlockOwnable {
     uint16 indexed version
   );
 
+  event SwapCall(
+    address lock,
+    address tokenAddress,
+    uint amountSpent
+  );
```

Comment:
One case I can think of to keep this event is to list all tokens that Unlock actually holds using the graph

Reply:
Ho that's a good idea in fact!